tests/forms_tests/tests/__init__.py

```diff
@@ -0,0 +1,23 @@
+from django.test.utils import override_settings
+
+
+def django_and_jinja_all_tests():
+    def decorate(cls):
+        for attr in cls.__dict__:
+            if attr.startswith('test'):
+                setattr(cls, attr, django_and_jinja(getattr(cls, attr)))
+        return cls
+    return decorate
+
+
+def django_and_jinja(func):
+    def test_django_and_jinja(*args, **kwargs):
+        with override_settings(
+            FORM_RENDERER='django.forms.renderers.Jinja2',
+            TEMPLATES={
+                'BACKEND': 'django.template.backends.jinja2.Jinja2',
+            },
+        ):
+            func(*args, **kwargs)
+        func(*args, **kwargs)
+    return test_django_and_jinja
```

Comment:
We could simplify this and also be explicit rather than assuming that the Django renderer is the default?

```suggestion
import inspect

from django.test.utils import override_settings


TEST_SETTINGS = [
    {
        'FORM_RENDERER': 'django.forms.renderers.DjangoTemplates',
        'TEMPLATES': {'BACKEND': 'django.template.backends.django.DjangoTemplates'},
    },
    {
        'FORM_RENDERER': 'django.forms.renderers.Jinja2',
        'TEMPLATES': {'BACKEND': 'django.template.backends.jinja2.Jinja2'},
    },
]


def test_all_form_renderers():

    def wrapper(func):
        def inner(*args, **kwargs):
            for settings in TEST_SETTINGS:
                with override_settings(**settings):
                    func(*args, **kwargs)
        return inner

    def decorator(cls):
        for name, func in inspect.getmembers(cls, inspect.isfunction):
            if name.startswith('test_'):
                setattr(cls, name, wrapper(func))
        return cls

    return decorator
```

(I've not tested this, but it should give you an idea.)